wolfcrypt/src/asn.c

```diff
@@ -11623,9 +11652,9 @@ static WC_INLINE int OurPasswordCb(char* passwd, int sz, int rw, void* userdata)
 int wc_KeyPemToDer(const unsigned char* pem, int pemSz,
                         unsigned char* buff, int buffSz, const char* pass)
 {
-    int            eccKey = 0;
-    int            ret;
-    DerBuffer*     der = NULL;
+    int ret;
+    int keyFormat = 0;
```

Comment:
Doesn't look like it's needed. keyFormat not used in function.
In PemToDer, when keyFormat is NULL then value not assigned.